### ISSUE TITLE
Underline annotations

### ISSUE THREAD

**Issue by mrtcode** [zotero/zotero#3258]
- [ ] Add support to pdf-worker.
- [ ] Add support to Zotero notes.
- [ ] Add support to note templates
- [ ] Allow to underline text from text selection popup (the one with 8 color swatches).
- [ ] Fix drag preview image.
- [ ] Draw it on page thumbnail and print output.

**Comment by dstillman** [comment]
> Allow to underline text from text selection popup 

@yexingsha can probably help with this.

**Comment by dstillman** [comment]
The idea there is that highlighting and underlining are functionally equivalent and most people will just use one or the other, so it should be possible to toggle between the two modes in that popup and have that be a persistent state with the same ability to just click a color to create a new annotation of the last-used type from the current text selection.

**Comment by mrtcode** [comment]
> The idea there is that highlighting and underlining are functionally equivalent and most people will just use one or the other, so it should be possible to toggle between the two modes in that popup and have that be a persistent state with the same ability to just click a color to create a new annotation of the last-used type from the current text selection.

Yes, I was thinking the same. We just need to figure out the UI part.

Although we already have an [issue](https://github.com/zotero/zotero/issues/3352) for this.
